packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/pages/home/index.jsx.hbs

```diff
@@ -82,7 +82,14 @@ const Home = () => {
             />
 
             <Hero
+                {{#if answers.project.demo.enableDemoSettings}}
+                title={intl.formatMessage({
+                    defaultMessage: 'The React PWA Starter Store for Retail',
+                    id: 'home.title.react_starter_store'
+                })}
+                {{else}}
```

Comment:
Small change here to make it more obvious in the demo site when locale is changed to another language.